recipes/example/meta.yaml

```diff
@@ -9,12 +10,13 @@
 #  `conda install openssl -c conda-forge``
 
 package:
-  name: simplejson
+  name: {{ name|lower }}
```

Comment:
Oh yeah you're totally right. I was thinking of the problem where the package name is `lazy-object-proxy` but what you want to import is `lazy_import_proxy`.